akka-remote/src/main/java/akka/remote/artery/AeronErrorLog.java

```diff
@@ -39,24 +35,20 @@
  */
 public class AeronErrorLog
 {
-    private final File cncFile;
     final MappedByteBuffer cncByteBuffer;
-    final DirectBuffer cncMetaDataBuffer;
-    final int cncVersion;
     final AtomicBuffer buffer;
 
-    public AeronErrorLog(File cncFile)
+    public AeronErrorLog(File cncFile, LoggingAdapter log)
```

Comment:
Please add `public AeronErrorLog(File cncFile) { this(cncFile, Logging.noLogging) }` to allow bin compat to be OK

Reply:
IntelliJ doesn't allow neither of the following:
- `Logging.noLogging`
- `Logging.NoLogging`

It is pulled from a Java class so not sure if it is a visibility issue, I also tried to get some `StdOutLogger` but couldn't find any, SOS :(

Reply:
By bad, it's just `akka.event.NoLogging.getInstance()` :)